src/artemis/v3/api/user/_get/getUsers.ts

```diff
@@ -0,0 +1,30 @@
+
+import ArtemisHeader from "../../../shared/ArtemisHeader"
+
+import { TResGetUsers } from "./TResGetUsers"
+
+
+/**
+ * get all users in a domain
+ * @description get all user in a domain
+ */
+export default async function getUsers(args: {
+    auth: {
+        token: string
+    }
+}): Promise<TResGetUsers> {
+    const headers = ArtemisHeader()
+    headers.append("Authorization", "Bearer " + args.auth.token)
+    const url = new URL(process.env.ARTEMIS_API + "/authentication-manager/user?page=0&size=1000&isActive=true&sort=updatedAt%2Cdesc&direction=desc")
+
+    console.log(">>> URL", url.toString())
```

Comment:
is it necessary? @ruztyA

Reply:
token is required,
filter in url i think only need params page, size, and isActive

console.log isn't necessary => I'll remove it
@regahr